apps/modernization-ui/src/apps/page-builder/page/management/preview/tab/question/PreviewQuestion.tsx

```diff
@@ -0,0 +1,106 @@
+import { PagesQuestion } from '../../../../../generated';
+import styles from './preview-question.module.scss';
+import { Input } from '../../../../../../../components/FormInputs/Input';
+import { Icon } from '@trussworks/react-uswds';
+import { SelectInput } from '../../../../../../../components/FormInputs/SelectInput';
+import { RadioButtons } from '../../../../../components/RadioButton/RadioButton';
+import { useEffect, useState } from 'react';
+import { Selectable } from '../../../../../../../options';
+import { ConceptOptionsResponse, ConceptOptionsService } from '../../../../../../../generated';
+import { authorization } from '../../../../../../../authorization';
```

Comment:
Suggestion: you can omit all of the `../../..` and start from the "top" down. The `tsconfig.json` has a `baseUrl` of `src`. 

So for example 
```
import { PagesQuestion } from '../../../../../generated';
```

can be 
```
import { PagesQuestion } from 'apps/page-builder/generated'
```